tests/database/test_database.py

```diff
@@ -0,0 +1,28 @@
+import pydantic
+import pytest
+from meowpi.database.mongo import *
```

Comment:
avoid using "*" on imports.
```suggestion
from meowpi.database.mongo import MongoDatabase
```